### ISSUE TITLE
multiple parser in one project

### ISSUE THREAD

**Issue by multics** [eslint/eslint#13247]
**Tell us about your environment**

* **ESLint Version:** 6.8.0
* **Node Version:** 14.0.0
* **npm Version:** 6.14.4

**What parser (default, Babel-ESLint, etc.) are you using?**
Here the problem comes.
babel-eslint, vue-eslint-parser, typescript-eslint-parser

What if my project needs two parsers to do lint work, how should I configure .eslintrc?

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js

```

```bash

```

**What did you expect to happen?**
Every parser is trying to replace others, since there is only one parser could be specified in the configure file. Is it possible to use all of them (maybe by some design changes)? Or one monopoly parser could do all work - I do not think it's possible.

**What actually happened? Please include the actual, raw output from ESLint.**
N/A

**Are you willing to submit a pull request to fix this bug?**
N/A

**Comment by kaicataldo** [comment]
Hi, you can use [glob based configuration](https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns) to achieve this by specifying the parser for a particular glob of files.
